SUPPORTED_APIS.md

```diff
@@ -85,7 +85,7 @@ The `status` options are:
 | [`EVM`](#evm-namespace) | [`evm_revert`](#evm_revert) | `SUPPORTED` | Revert the state of the blockchain to a previous snapshot |
 | `EVM` | `evm_setAccountBalance` | `NOT IMPLEMENTED` | Sets the given account's balance to the specified WEI value |
 | `EVM` | `evm_setAccountCode` | `NOT IMPLEMENTED` | Sets the given account's code to the specified data |
-| `EVM` | `evm_setAccountNonce` | `NOT IMPLEMENTED` | Sets the given account's nonce to the specified value |
+| [`EVM`](#evm-namespace) | [`evm_setAccountNonce`](#evm_setaccountnonce) | `SUPPORTED` | Sets the given account's nonce to the specified value |
```

Comment:
shouln't we point here that we support `anvil_` as well?

Reply:
Hmm, yes, I've added the aliases. But I wonder: will the links work? Is there an #anvil-namespace anchor in the target, or should it be added? Where are the links pointing to, anyway?

Reply:
1.(side comment but important to note) Please, refactor the table. It has to be structured. There are bunch of namespaces we support( I am thinking about splitting tables by namespace `eth`, `zks`,  etc.
2. for now just add to the table(as was mention in the first point - create new table for `anvil_` namespace),  without linking.

Reply:
So, add to the table, or create new table? For now, I've just removed the links (so there's still 1 table).

Reply:
There should be 1 table for specific namespace  endpoints.
one for `eth_`
then one for `zks_`
and so on

Reply:
but lets do it in separate PR

Reply:
please, if you leave the link (you are doing `(#evm_setaccountnonce) `) here -  add description below in this file for this endpoint like this:
<img width="1319" alt="image" src="https://github.com/user-attachments/assets/834dbac5-5dae-47f1-9ebe-d4eb4de543f3">